pkg/storage/multi.go

```diff
@@ -27,11 +28,18 @@ func ReadTee(ctx context.Context, sStore Store, source string, dStore Store, des
 	return object, err
 }
 
+// MultiStoreUnit is used to specify multiple operations, some of which are tolerated to fail
 type MultiStoreUnit struct {
-	Store           Store
+	// Store is the backend to be accessed
+	Store Store
+
+	// TolerateFailure to false breaks multi-store operations whenever an error is encountered.
 	TolerateFailure bool
 }
 
+// MultiPut duplicates write operations to an array of stores, under the same name
+//
+// TODO(fred): the aliased type NewKey is just confusing: fallback to bool
 func MultiPut(ctx context.Context, stores []MultiStoreUnit, name string, buffer []byte, doesNotExist NewKey) error {
```

Comment:
FYI: This was added to allow for a local cache to which the same data is written to as the remote CAFS.. This was when we were still chasing k8s CSI integration.

Reply:
@kerneltime ok got it.
My remark was specifically about using a type alias here, which is basically pointless.
Type aliases are not the way to construct enums in go, if this was the initial intent here. Moreover, there is no real point in constructing an enum for a boolean value.

My point was just to replace this by: 
`func MultiPut(ctx context.Context, stores []MultiStoreUnit, name string, buffer []byte, doesNotExist bool) error`

Reply:
will do later